packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/TriggerConditions.tsx

```diff
@@ -0,0 +1,208 @@
+import { Autocomplete, Box, TextField, Typography } from '@linode/ui';
+import { Grid } from '@mui/material';
+import * as React from 'react';
+import { Controller, useFormContext, useWatch } from 'react-hook-form';
+
+import {
+  EvaluationPeriodOptions,
+  PollingIntervalOptions,
+} from '../../constants';
+
+import type { CreateAlertDefinitionForm } from '../types';
+import type {
+  CreateAlertDefinitionPayload,
+  TriggerCondition,
+} from '@linode/api-v4';
+import type { FieldPathByValue } from 'react-hook-form';
+interface TriggerConditionProps {
+  /**
+   * maximum scraping interval value for a metric to filter the evaluation period and polling interval options
+   */
+  maxScrapingInterval: number;
+  /**
+   * name used for the component to set in form
+   */
+  name: FieldPathByValue<CreateAlertDefinitionPayload, TriggerCondition>;
+}
+export const TriggerConditions = React.memo((props: TriggerConditionProps) => {
+  const { maxScrapingInterval, name } = props;
+
+  const { control } = useFormContext<CreateAlertDefinitionForm>();
+  const serviceTypeWatcher = useWatch({
+    control,
+    name: 'serviceType',
+  });
+  const getPollingIntervalOptions = () => {
+    const options = serviceTypeWatcher
+      ? PollingIntervalOptions[serviceTypeWatcher]
+      : [];
+    return options.filter((item) => item.value >= maxScrapingInterval);
+  };
+
+  const getEvaluationPeriodOptions = () => {
+    const options = serviceTypeWatcher
+      ? EvaluationPeriodOptions[serviceTypeWatcher]
+      : [];
+    return options.filter((item) => item.value >= maxScrapingInterval);
+  };
+
+  return (
+    <Box
+      sx={(theme) => ({
+        backgroundColor:
+          theme.name === 'light' ? theme.color.grey5 : theme.color.grey9,
```

Comment:
are they fine with theme.color colors or Linode team have specific color pallets for this?